src/models/editorArguments.interface.ts

```diff
@@ -2,8 +2,9 @@
 import { Column, ElementPosition, PositionMethod } from './index';
 import type { SlickDataView } from '../slick.dataview';
 import type { SlickGrid } from '../slick.grid';
+import type { Column, ElementPosition, GridOption, PositionMethod } from './index';
```

Comment:
more duplicate imports since line 2 and 5 are nearly the same, line 5 seems better to keep since it adds the missing `type` which is good

Reply:
I am sorry - I did not run a build since I accepted your changes in the browser only and did not sync.

Reply:
np